Source/Core/VideoBackends/Vulkan/VKTexture.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include "VideoBackends/Vulkan/VKTexture.h"
 
+#include <VideoCommon/DriverDetails.h>
 #include <algorithm>
```

Comment:
```suggestion
#include "VideoCommon/DriverDetails.h"

#include <algorithm>
```

Reply:
Shouldn't it be this instead?

```suggestion
#include <algorithm>

#include "VideoCommon/DriverDetails.h"
```

Reply:
Oh, whoops. Yes, it should go below all of the <> includes, next to `#include "VideoCommon/VideoConfig.h"`.